README.md

```diff
@@ -77,6 +77,7 @@ The tooltip wraps an element _in place_ in your React Native rendering. When it
 | closeOnChildInteraction | bool | true | When child interaction is allowed, this prop determines if `onClose` should be called when the user interacts with the child element. Default is true (usually means the tooltip will dismiss once the user touches the element highlighted) |
 | closeOnContentInteraction | bool | true | this prop determines if `onClose` should be called when the user interacts with the content element. Default is true (usually means the tooltip will dismiss once the user touches the content element) |
 | content          | function/Element | `<View />`                             | This is the view displayed in the tooltip popover bubble                                                                                                                                                       |
+| contentStyle          | CSS |                             | Custom CSS to modify the tooltip as needed.                                                                                                                                                       |
```

Comment:
The type should be `StyleProp<ViewStyle>` and the explanation should match with it. 

Because in react native, we're not using CSS, it is its own style